File: PUNTOS DE VENTA/puntoDeVenta/puntoDeVenta/Classes/IGraphicalAddon.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Drawing;
using System.Windows.Forms;
using System.Data;
namespace puntoDeVenta.Classes
{
    public interface IGraphicalAddon
    {
        String Name { get; }
        String NameText { get; }
        String Category { get; }
        String idProduct { get; }
        Boolean mdiStatus { get; }
        Image Image { get; }
        Form ParentForm { set; }
        Object SaycSetting { set; }
        DataSet DataSource { set; }
        void OnClick(Object sender, System.EventArgs e);
    }
}

File: Usuarios_Sin terminar/FormPrueba/Classes/IGraphicalAddon.vb
﻿Public Interface IGraphicalAddon
    ReadOnly Property Name As String
    ReadOnly Property NameText As String
    ReadOnly Property Category As String
    ReadOnly Property idProduct As String
    ReadOnly Property mdiStatus As Boolean
    ReadOnly Property Image As Image
    WriteOnly Property ParentForm As Form
    WriteOnly Property SaycSetting As Object
    WriteOnly Property DataSource As DataSet
    Sub OnClick(ByVal sender As Object, ByVal e As System.EventArgs)
End Interface